examples/olsconfig.yaml

```diff
@@ -21,9 +21,13 @@ OLSConfig:
     library_log_level: warning
   default_provider: bam
   default_model: ibm/granite-13b-chat-v2
+Authentication:
+    ols_auth_check: False
+    k8s_cluster_api: "https://api.example.com:6443"
+    k8s_ca_cert_path: "/Users/home/ca.crt"
+    verify_ssl: False
```

Comment:
name this `skip_tls_verification` with a default of False so that the default behavior is that you get verification.

also ssl is an outdated term, it refers to an older version of TLS, so TLS is the preferred terminology.
```
Transport Layer Security (TLS) is the upgraded version of SSL that fixes existing SSL vulnerabilities. 
```